src/lib/repoStatusCode.test.js

```diff
@@ -0,0 +1,15 @@
+import {repoStatusCode} from "./repoStatusCode";
+
+describe("Test: repoStatusCode()", () => {
+  test("github/gh-ost should return 200", async() => {
+    const repoUrl = "github/gh-ost";
+    const result = await repoStatusCode(repoUrl);
+    expect(result).toBe(200);
+  });
+
+  test("github/ghost should return 404", async() => {
+    const repoUrl = "github/ghost";
+    const result = await repoStatusCode(repoUrl);
+    expect(result).toBe(404);
+  });
```

Comment:
One more check is 301. i.e. bdougie/open-sauced will return "moved permanently" status.

Reply:
@bdougie It is returning `200` :thinking: 

![Screenshot](https://user-images.githubusercontent.com/61746440/85225010-c7dc2000-b3eb-11ea-827f-3f0958707277.png)

Reply:
In **curl** it returns `301`

![Screenshot](https://user-images.githubusercontent.com/61746440/85225102-65375400-b3ec-11ea-8527-0e341b1ec317.png)

Reply:
@bdougie It works in `AddRepoForm` but not in test file!

![Screenshot](https://user-images.githubusercontent.com/61746440/85225636-5d79ae80-b3f0-11ea-8572-c761b9932ae9.png)

Reply:
Well, that is interesting. It must have something to do with the test environment. I am fine to omit the test for now. Could you add a TODO or comment stating this? After the 1.0 launch, I would love to circle back to the testing issue and find solutions.